### PR TITLE
Debug updates

### PR DESCRIPTION
**Pull request by rodrigosiqueira**

### CODE REVIEW

**Comment by rodrigosiqueira** [comment]
@kwy95 could you review this PR? I added a new tutorial; it would be nice to have your review in that specific part.

File: documentation/tutorials/kernel-debug.rst [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+Save (`--history | -k`)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This option will create a folder named `kw_debug` in the place that you run
+this command and a sub-folder that follow this pattern::
+
+  <ID>_<YEAR-MONTH-DAY>
+
+In a few words, every new execution is saved in that folder.
```
Comment (kwy95) [inline documentation/tutorials/kernel-debug.rst:84]: Do the folders contain any files? If so, which ones? It isn't clear to me why I'd want to use this option.

File: documentation/tutorials/kernel-debug.rst [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+Command (`--cmd | -c`)
+~~~~~~~~~~~~~~~~~~~~~~
+
+This option expects a parameter which is a command to be executed in the target
+machine. The idea is something like this:
+
+#. Clean the target log
+#. Start to collect log
+#. Run the command
+#. Stop to collect log
```
Comment (kwy95) [inline documentation/tutorials/kernel-debug.rst:95]:
```suggestion
#. Start logging
#. Run the command
#. Stop logging
```

File: documentation/tutorials/kernel-debug.rst [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+     11  amdgpu_dm_crtc_atomic_check
+     12  amdgpu_dm_dc_clocks_state
+     13  amdgpu_dm_dce_clocks_state
+     14  amdgpu_dm_dc_pipe_state
+     15  amdgpu_dm_plane_atomic_check
+     16  amdgpu_dmub_trace_high_irq
+     17  amdgpu_refresh_rate_track
+     18  dcn_fpu
+
+Notice that you can put any specific event after the `events:`. Finally, let's
```
Comment (kwy95) [inline documentation/tutorials/kernel-debug.rst:161]:
```suggestion
Notice that you can put any specific event after `events:`. Finally, let's
```

File: documentation/tutorials/kernel-debug.rst [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+ 3. mmiotrace
+ 4. function_graph
+ 5. wakeup_dl
+ 6. wakeup_rt
+ 7. wakeup
+ 8. function
+ 9. nop
+
+Those are the ftrace algorithm that you can set; probably, function_graph is
+the most common one, and you can use it like this::
```
Comment (kwy95) [inline documentation/tutorials/kernel-debug.rst:195]:
```suggestion
Those are the ftrace algorithms that you can set; function_graph is probably
the most common one, and you can use it like this::
```

**Review by kwy95 — CHANGES_REQUESTED** [review]
Apart from the comments I left some of the commits have no messages in them, only the titles.

**Comment by rodrigosiqueira** [comment]
Merged to the unstable branch.

**Pull request closed without merging**